.gitignore

```diff
@@ -1,6 +1,7 @@
 /build
 /vendor
 
+.idea/
```

Comment:
This is not library specific and should go into your global `.gitignore`. Could you please remove this again?